2024/json-ld-wg/index.html

```diff
@@ -225,10 +256,20 @@ <h3>
             <ul>
               <li>Application profile of JSON-LD to enable efficient streaming parsers.</li>
               <li>Test suites and implementation reports for the specifications.</li>
-              <li>JSON-LD 1.1 specified in other serializations like YAML or CBOR.</li>
               <li>Best practices for use and implementation of JSON-LD 1.1.</li>
             </ul>
         </section>
+
+        <section id="timeline">
+          <h3>Timeline</h3>
+            <ul>
+              <li>September 2022: First teleconference</li>
+              <li>Q2 2024: FPWD of CBOR-LD and YAML-LD</li>
+              <li>Q4 2024: Candidate Recommendation of CBOR-LD and YAML-LD</li>
+              <li>Q1 2025: Candidate Recommendation of CBOR-LD and YAML-LD</li>
```

Comment:
```suggestion
              <li>Q4 2024: Candidate Recommendation of YAML-LD</li>
              <li>Q1 2025: Candidate Recommendation of CBOR-LD</li>
```

Reply:
the intention was in fact to have CR of both in Q4 2024, and *Recommendation* of both on Q1 2025. I'll fix it right away.